pkg/minikube/update/update.go

```diff
@@ -43,24 +43,26 @@ import (
 )
 
 const (
-	updateLinkPrefix   = "https://github.com/jimmidyson/minishift/releases/tag/" + version.VersionPrefix
-	downloadLinkFormat = "https://github.com/jimmidyson/minishift/releases/download/v%s/%s"
-	githubOwner        = "jimmidyson"
-	githubRepo         = "minishift"
-	timeLayout         = time.RFC1123
+	timeLayout  = time.RFC1123
+	githubOwner = "jimmidyson"
+	githubRepo  = "minishift"
 )
 
 var (
 	lastUpdateCheckFilePath = constants.MakeMiniPath("last_update_check")
 	githubClient            *github.Client
-	downloadBinary          = "minishift-" + runtime.GOOS + "-" + runtime.GOARCH
 )
 
 func MaybeUpdateFromGithub(output io.Writer) {
 	MaybeUpdate(output, githubOwner, githubRepo, lastUpdateCheckFilePath)
 }
 
 func MaybeUpdate(output io.Writer, githubOwner, githubRepo, lastUpdatePath string) {
+
+	downloadBinary := githubRepo + "-" + runtime.GOOS + "-" + runtime.GOARCH
```

Comment:
Binary name will not always be githubRepo - can you make binary name configurable please.